packages/core/src/eth/utils.ts

```diff
@@ -32,8 +32,8 @@ export const convertTransactionDTO = (
         }
       : null,
     rawTransaction: convertRawTransactionDTO(rawTransaction),
-    fee: transactionDTO.fee,
-    estimatedFee: transactionDTO.estimatedFee,
+    fee: BNConverter.hexStringToBN(transactionDTO.fee),
```

Comment:
```suggestion
    fee: BNConverter.hexStringToBN(String(transactionDTO.fee)),
```
안전하게 String으로 한번 감싸주면 좋을 것 같습니다!

Reply:
수정했습니다! 👍